Jenkinsfile

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env groovy
+
+def config = jobConfig {
+    cron = '@midnight'
+    nodeLabel = 'docker-oraclejdk8'
+    testResultSpecs = ['junit': '**/build/test-results/**/TEST-*.xml']
+    slackChannel = '#kafka-warn'
+    timeoutHours = 4
+    runMergeCheck = false
+}
+
+
+def job = {
+    // https://github.com/confluentinc/common-tools/blob/master/confluent/config/dev/versions.json
+    def kafkaMuckrakeVersionMap = [
+            "2.3": "5.3.x",
```

Comment:
We need to add the mapping for 2.2 and other older branches here.

Reply:
Done. New build triggered.

Reply:
we should probably have this managed from a central location wrapped by a dev tool cli, eventually, rather than having to copy this across the branches.